libs/core/src/lib/vertical-navigation/vertical-navigation.component.ts

```diff
@@ -21,16 +24,86 @@ export class VerticalNavigationComponent implements AfterContentInit {
     @Input()
     condensed = false;
 
+    /** An RxJS Subject that will kill the data stream upon component’s destruction (for unsubscribing)  */
+    private readonly _onDestroy$: Subject<void> = new Subject<void>();
```

Comment:
minor: `private readonly _onDestroy$ = new Subject<void>()`, type will be inferred.